README.md

```diff
@@ -1,213 +1,229 @@
-# Autoscaling Kubernetes on GKE
+# Autoscaling Qlik Core on Google Kubernetes Engine
+
+This use case shows how you can set up a Qlik Core application in a Google Kubernetes Engine (GKE) cluster so you can easily scale your application up or down to meet user demands.
 
 ## Prerequisites
 
-* Setup GKE (Google Kubernetes Engine) by following this guide: https://cloud.google.com/kubernetes-engine/docs/quickstart
+* Set up GKE by following the GKE [Quickstart guide](https://cloud.google.com/kubernetes-engine/docs/quickstart).
+
+* Modify the `settings.config` for your project.
+
+    !!! Note
+        You must add your project name after `:-`.
 
-* Modify the `settings.config` for your needs. Do note that you **have to add your Project name**. Example: `GCLOUD_PROJECT="${GCLOUD_PROJECT:-YOUR-PROJECT-HERE}"` add your project name after `:-` you also **have** to add your license serial number and your license control number.
+        Example: `GCLOUD_PROJECT="${GCLOUD_PROJECT:-YOUR-PROJECT-HERE}"`
```

Comment:
I think we should add that you need to specify your license in the `settings.config` file, otherwise it wont work if you do the "all at once command"